### ISSUE TITLE
This library has GPL dependency

### ISSUE THREAD

**Issue by ypresto** [glebm/i18n-tasks#234]
This library depends on term-ansicolor.

In Gemfile:

```ruby
s.add_dependency 'term-ansicolor', '>= 1.3.2'
```

In code:

```ruby
def warn_deprecated(message)
  log_stderr Term::ANSIColor.yellow Term::ANSIColor.bold "#{program_name}: [DEPRECATED] #{message}"
end
```

And term-ansicolor is released under GPLv2.

It means this library should be licensed under GPL or drop dependency to term-ansicolor.
https://www.gnu.org/licenses/gpl-faq.en.html#MereAggregation

**Comment by ypresto** [comment]
Someone attempted to ask to change license but it failed... See issue 24 of term-ansicolor.

**Comment by ypresto** [comment]
Possible alternative to term-ansicolor: https://github.com/fazibear/colorize

**Comment by ypresto** [comment]
Possible alternative licensed under MIT:
https://github.com/sickill/rainbow

**Comment by glebm** [comment]
Well, ouch. Accepting PRs to switch to Rainbow, and left a comment on issue 24 of term-ansicolor.

**Comment by ypresto** [comment]
rainbow does not have `dark` method used in i18n-tasks so I sent PR to add it:
https://github.com/sickill/rainbow/pull/53

**Comment by ypresto** [comment]
Also added uncolor replacement:
https://github.com/sickill/rainbow/pull/54

**Comment by ypresto** [comment]
As [license_finder](https://github.com/pivotal/LicenseFinder), current (0d31ef467636d92a1fe08802c86c1a3c2f043285) library licenses are:

```
Dependencies that need approval:
activesupport, 5.0.2, MIT
ast, 2.3.0, MIT
axlsx, 2.0.1, MIT
bundler, 1.13.6, unknown
byebug, 9.0.6, "Simplified BSD"
concurrent-ruby, 1.0.5, MIT
diff-lcs, 1.3, "MIT, Artistic-2.0, GPL-2.0+"
docile, 1.1.5, MIT
easy_translate, 0.5.0, unknown
erubis, 2.7.0, MIT
highline, 1.7.8, ruby
htmlentities, 4.3.4, MIT
httparty, 0.14.0, MIT
i18n, 0.8.1, MIT
i18n-tasks, 0.9.12, MIT
json, 2.0.3, ruby
license_finder, 3.0.0, MIT
mini_portile2, 2.1.0, MIT
minitest, 5.10.1, MIT
multi_xml, 0.6.0, MIT
nokogiri, 1.7.1, MIT
parser, 2.4.0.0, MIT
powerpack, 0.1.1, MIT
rainbow, 2.2.1, MIT
rake, 12.0.0, MIT
rspec, 3.5.0, MIT
rspec-core, 3.5.4, MIT
rspec-expectations, 3.5.0, MIT
rspec-mocks, 3.5.0, MIT
rspec-support, 3.5.0, MIT
rubocop, 0.48.0, MIT
ruby-progressbar, 1.8.1, MIT
rubyzip, 1.0.0, ruby
simplecov, 0.14.1, MIT
simplecov-html, 0.10.0, MIT
terminal-table, 1.7.3, MIT
thor, 0.19.4, MIT
thread, 0.2.2, WTFPL
thread_safe, 0.3.6, "Apache 2.0"
tzinfo, 1.2.3, MIT
unicode-display_width, 1.1.3, MIT
with_env, 1.1.0, MIT
xml-simple, 1.1.5, ruby
yard, 0.9.8, MIT
```

LGTM..!

**Comment by glebm** [comment]
Released [v0.9.13](https://github.com/glebm/i18n-tasks/releases/tag/v0.9.13)!

**Comment by ypresto** [comment]
Thanks, applied to our project and working fine..! :tada:
